10-slots/03-UiCalendarView/components/MeetupsCalendar.vue

```diff
@@ -1,16 +1,29 @@
 <template>
-  <UiCalendarView>
-    <UiCalendarEvent v-if="meetups[0]" tag="a" :href="`/meetups/${meetups[0].id}`">
-      {{ meetups[0].title }}
+  <UiCalendarView v-slot="{ day, year, month }">
+    <UiCalendarEvent
+      v-for="meetup in getDateMeetups(day, month, year)"
+      tag="a"
+      :href="`/meetups/${meetup.id}`"
+      :key="meetup"
+    >
+      {{ meetup.title }}
     </UiCalendarEvent>
   </UiCalendarView>
```

Comment:
Calling methods in a Vue template is a bad practice in general. It's ok when it is a fast simple method of output formatting. But `getDateMeetups` is a data processing method. It is moving component's logic, a data processing logic, to the template.

Also, it is not good for performance. On each component re-render, for rendering each cell, it makes a full iteration over the entire array. 

Instead, it's better to prepare data for optimal and simple rendering. For example, creating an object, map from date to meetup on this date.